bin/launcher.ps1

```diff
@@ -87,6 +95,17 @@ function AddClassPathRequiredForExecution() {
   }
 }
 
+function EscapeSpaceInClasspath {
+  $classpath = $global:classpath.Split(";")
+  $global:classpath = ""
+  foreach ($cp in $classpath.Split(";")) {
+      if ($cp.Contains(" ") ) {
+        $cp = "`"$cp`"" -replace '\\','\\'
```

Comment:
what's `-replace` doing here? I see that the original and replacement are same text?

Reply:
Not exactly. It's replacing `\` to `\\`. This patter looked weird to me as well but somehow, frist arg needs to escape but not the second one.

Here is the example snippet
```pwsh
PS /> $a = "C:\foo\bar"
PS /> $a -replace "\","\\"
The regular expression pattern \ is not valid.
At line:1 char:1
+ $a -replace "\","\\"
+ ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (\:String) [], RuntimeException
+ FullyQualifiedErrorId : InvalidRegularExpression

PS /> $a -replace "\\","\\"
C:\\foo\\bar
```